### PR TITLE
[Azure Stack] Updates to Azs.Backup.Admin spec

### PR DESCRIPTION
**Pull request by viananth**
commit 72efdec25338cfbb9fea108dcfb390f4c5ff6295
Author: bganapa <bganapa@microsoft.com>
Date:   Thu Feb 27 17:26:56 2020 -0800

    Update to have $(this-folder)/ in the input files

commit 10ec2068533a0b5d1f5641b85030678ef61ebf75
Author: Daotian Zhang <38281880+daozha@users.noreply.github.com>
Date:   Fri Jan 3 04:01:04 2020 +0800

    Add x-ms-long-running-operation-options (#8037)

commit 70fbb623c3f2d965cd33c282da7439fe5262e2b8
Author: Daotian Zhang <38281880+daozha@users.noreply.github.com>
Date:   Wed Dec 18 14:13:52 2019 +0800

    Fix resourceGroups in resource path and add top/skip (#7996)

commit 9e551f0eab4057d4c2f54c333c7aa2a1a564c125
Author: bganapa <bganapa@microsoft.com>
Date:   Tue Nov 12 11:44:22 2019 -0800

    Reset to Stackadmin2 (#7766)

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by bganapa — APPROVED** [review]

**Comment by bganapa** [comment]
@daozha Could you pls review approve this PR?

File: specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/stable/2016-05-01/Backup.json [hunk 1]
```diff
@@ -0,0 +1,204 @@
+{
```
Comment (daozha) [inline specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/stable/2016-05-01/Backup.json:1]: Could you remove all files under stable/2016-05-01? This API version has been deprecated.

File: specification/azsadmin/resource-manager/backup/readme.md [hunk 2]
```diff
@@ -38,6 +38,17 @@ input-file:
...
+### Tag: package-2016-05-01
+
+These settings apply only when `--tag=package-2016-05-01` is specified on the command line.
+
+``` yaml $(tag) == 'package-2016-05-01'
+input-file:
+    - $(this-folder)/Microsoft.Backup.Admin/stable/2016-05-01/Backup.json
+    - $(this-folder)/Microsoft.Backup.Admin/stable/2016-05-01/BackupLocations.json
+    - $(this-folder)/Microsoft.Backup.Admin/stable/2016-05-01/Backups.json
+```
```
Comment (daozha) [inline specification/azsadmin/resource-manager/backup/readme.md:50]: Could you also remove these lines as 2016-05-01 has been removed

File: specification/azsadmin/resource-manager/backup/readme.azsautogen.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Backup Admin
```
Comment (daozha) [inline specification/azsadmin/resource-manager/backup/readme.azsautogen.md:1]: Just curious, what's the difference between this azsautogen one and the original readme.md?

**Review by daozha — CHANGES_REQUESTED** [review]
@viananth , could you remove all those related to API version 2016-05-01, which has been deprecated? Thanks!

File: specification/azsadmin/resource-manager/backup/readme.azsautogen.md (hunk 3 above)
Comment (viananth) [reply specification/azsadmin/resource-manager/backup/readme.azsautogen.md]: It was meant to carry the autorest gen info. Now all the info have been moved to readme.md. and removed azsautogen.md

Comment (daozha) [reply specification/azsadmin/resource-manager/backup/readme.azsautogen.md]: Got it. Then do we need to update PowerShell generation side also? https://github.com/Azure/azurestack-powershell/blob/master/src/Azs.Backup.Admin/readme.md
Currently this points to readme.azsautogen.md.

Actually I'm also fine if we have this azsautogen.md, as long as we align with other Azs RPs.

**Comment by daozha** [comment]
For the validation errors, most are about the breaking changes that I changed paths containing "resourcegroups" to "resourceGroups", which is expected. Also there's a complain about formatting. I'll take a look today and update.

**Comment by daozha** [comment]
@viananth , not sure why but I could not find your fork to send PR. So could you cherry pick this commit directly to your branch? It should help fix the PrettierCheck errors:
https://github.com/daozha/azure-rest-api-specs/commit/c57e0aafc92464d0588273a1ec5a03c53bf573fe.

The BreakingChange errors are about changing paths containing "resourcegroups" to "resourceGroups", and such change is expected.

**Comment by viananth** [comment]
Thank you @daozha ! I have cherry-picked your commit.

**Review by daozha — APPROVED** [review]
@viananth , @bganapa , the PR looks good to me. The remaining validation errors come from breaking changes where I changed "resourcegroup" to "resourceGroup". I think this is expected.

As the file readme.azsautogen.md is removed, we might need to also update the PowerShell generation side which points to azsautogen.md: https://github.com/Azure/azurestack-powershell/blob/master/src/Azs.Backup.Admin/readme.md

**Pull request merged**